spec/uploader_spec.rb

```diff
@@ -23,7 +23,9 @@
   end
 
   it "should successfully upload a file from IO" do
-    result = Cloudinary::Uploader.upload(File.open(TEST_IMG, "rb"), :tags => [TEST_TAG, TIMESTAMP_TAG])
+    File.open do |file|
```

Comment:
Sorry, I'm clearly having one of those days. You need to specify the file to open. `File.open(TEST_IMG, "rb") do |file|` etc.